internal/web/context.go

```diff
@@ -0,0 +1,22 @@
+package web
+
+import (
+	"github.com/labstack/echo/v4"
+	"go.uber.org/zap"
+)
+
+type AppContext struct {
```

Comment:
Using `zap.Logger` directly in `AppContext` can lead to tight coupling with the logging library. If the goal is to maintain flexibility and make testing easier, consider defining an interface for logging instead.